src/sdk/sdk.ts

```diff
@@ -352,7 +352,9 @@ export class FriendlyCaptchaSDK {
         send({ type: "root_destroy_widget" });
         this.bus.removeTarget(widgetId);
         this.widgets.delete(widgetId);
-        opts.element.remove();
+
+        opts.element.innerHTML = "";
```

Comment:
I believe this may trigger CSP setups without `unsafe-inline`  (but I'm not entirely sure: you could validate by writing a test for it in `sdktest`). If it does pose a problem, I imagine we can loop over the children and remove them one by one.